src/legacy/ManifestTranslator.ts

```diff
@@ -309,10 +308,6 @@ export class ManifestTranslator {
       application.api = Object.assign(new ManifestApi(), app.api);
     }
 
-    if (app.configuration) {
-      application.configuration = app.configuration.map((item) => Object.assign(new ConfigurationItem(), item));
```

Comment:
this looks like an unrelated change. It has been there since 2022 there are apps using config feature we must make sure we don't break anything.